stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -366,6 +389,140 @@ public struct Unsafe${Mutable}BufferPointer<Element>
   public func makeIterator() -> UnsafeBufferPointerIterator<Element> {
     return UnsafeBufferPointerIterator(_position: _position, _end: _end)
   }
+  
+  /// Deallocates the heap block previously allocated at this buffer pointer’s 
+  /// base address. 
```

Comment:
I don't think we're using "heap block" anywhere else in these docs. Can you use "memory region" or "memory block" instead?